W09H01/src/test/pgdp/hashset/TupleHashSetTest.java

```diff
@@ -415,14 +415,5 @@ void iteratorFullHashSet() {
       tuplesInHashSet.add(integerIntegerTuple);
     }
     assertEquals(353, tuplesInHashSet.size());
-    Arrays.stream(tuples).forEach(integerIntegerTupleHashSet::insert);
-    
-    tuplesInHashSet = new ArrayList<>();
-    for (Tuple<Integer, Integer> integerIntegerTuple : integerIntegerTupleHashSet) {
-      tuplesInHashSet.add(integerIntegerTuple);
-    }
-    assertEquals(353, tuplesInHashSet.size());
-    
```

Comment:
This is simply code duplication, right?

Reply:
Yeah it is duplicate code, but as the HashSet is not cleared in between the "instert-blocks", this should check, if the iterator does not change the saved data.

Reply:
Ah, right! I did not see that! I'll close this pull request :+1: